civictechprojects/caching/cache.py

```diff
@@ -134,6 +134,8 @@ def _projects_tag_counts(event=None, group=None):
                 organization_type += project.project_organization_type.slugs()
 
                 project_positions = project.get_project_positions()
+                # exclude roles which are hidden
+                project_positions = filter(lambda position: not position.is_hidden, project_positions)                
```

Comment:
This technically works, but our preferred way of filtering is to leverage django's ORM.  

```suggestion
                project_positions = project.get_project_positions().filter(is_hidden=False)            
```

Reply:
Updated the code and tested